catalog/spatial/registry/registry-federation-admin-service-impl/src/main/java/org/codice/ddf/registry/federationadmin/service/impl/IdentityNodeInitialization.java

```diff
@@ -104,6 +111,30 @@ public void destroy() {
         executorService.shutdown();
     }
 
+    private void updateIdentityNodeName(Metacard metacard) throws FederationAdminException {
+        RegistryPackageType registryPackage = federationAdminService.getRegistryObjectByRegistryId(
```

Comment:
Since we already have the metacard we can just just the MetacardMarshaller's getRegistryPackageFromMetacard(metacard) instead of making another catalog call.